src/app/story/story.component.ts

```diff
@@ -108,30 +118,31 @@ export class StoryComponent implements OnInit, AfterViewInit, OnDestroy {
       // Don't show the connecting lines until the whole story has been processed
       jsPlumb.setSuspendDrawing(true);
 
+
       // Loop through all the story rows except for the last one, because that
       // relationship flows backwards and is set after the loop
       for (let i = 0; i < storyRows.length - 1; i++) {
         // Connect the main node to the next main node
         const nextNode = storyRows[i + 1].querySelector('.node-column').children[0];
         jsPlumb.connect({
           source: storyRows[i].querySelector('.node-column').children[0],
-          target: nextNode
+          target: nextNode,
```

Comment:
I have no problems with adding this comma, but it's unrelated to the commit.

Reply:
Oops! I had something else in there on the following line and then later deleted it, but must have forgotten to take the comma off with it.